net-mgmt/pfSense-pkg-snmptt/files/usr/local/pkg/snmptt.xml

```diff
@@ -0,0 +1,730 @@
+					logs to ALL of the enabled log systems.  Warning:  If multiple log systems are<br/>
+					enabled and only one fails, the other log system will continuously be logged to<br/>
+					until ALL of the log systems function.<br/>
+					The recommended setting is 1 with only one log system enabled.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+				<option><name>2</name><value>2</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Duplicate Trap Window</fielddescr>
+			<fieldname>duplicate_trap_window</fieldname>
+			<description>How often in seconds duplicate traps will be processed.</description>
+			<default_value>0</default_value>
+			<type>input</type>
+			<size>3</size>
+			<required>true</required>
+		</field>
+		<field>
+			<name>Logging</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>Stdout Enable</fielddescr>
+			<fieldname>stdout_enable</fieldname>
+			<description>Set to 1 to enable messages to be sent to standard output, or 0 to disable.</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Log Enable</fielddescr>
+			<fieldname>log_enable</fieldname>
+			<description>Set to 1 to enable text logging of TRAPS.</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Log File</fielddescr>
+			<fieldname>log_file</fieldname>
+			<description>Log file location. The COMPLETE path and filename.</description>
+			<default_value>/var/log/snmptt/snmptt.log</default_value>
+			<type>input</type>
+			<size>150</size>
+		</field>
+		<field>
+			<fielddescr>Log System Enable</fielddescr>
+			<fieldname>log_system_enable</fieldname>
+			<description>Set to 1 to enable text logging of SNMPTT system errors.</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Log System File</fielddescr>
+			<fieldname>log_system_file</fieldname>
+			<description>Log file location. The COMPLETE path and filename.</description>
+			<default_value>/var/log/snmptt/snmpttsystem.log</default_value>
+			<type>input</type>
+			<size>150</size>
+		</field>
+		<field>
+			<fielddescr>Unknown Trap Log Enable</fielddescr>
+			<fieldname>unknown_trap_log_enable</fieldname>
+			<description>Set to 1 to enable logging of unknown traps.</description>
```

Comment:
Put short descriptions of these behaviors in the "name" part of the options list.